lib/check-port.js

```diff
@@ -0,0 +1,31 @@
+const tcpPortUsed = require('tcp-port-used');
+const chalk = require('chalk');
+
+function checkPort(port, content, numOfAttempts, MAX_ATTEMPTS) {
+  tcpPortUsed
+    .check(port, 'localhost')
+    .then(function(inUse) {
+      if (inUse && numOfAttempts >= MAX_ATTEMPTS) {
+        console.log(
+          'Reached max attempts, exiting. Please open up some ports or ' +
+            'increase the number of attempts and try again.'
+        );
+        process.exit(1);
+      } else if (inUse) {
+        console.error(chalk.red('Port ' + port + ' is in use'));
+        // Try again but with port + 1
```

Comment:
remove the comment. the code is self explanatory

Reply:
Sure 👍